diff --git a/l10n_br_hr/models/hr_employee.py b/l10n_br_hr/models/hr_employee.py
index b67196e1d2..924f4a9675 100644
--- a/l10n_br_hr/models/hr_employee.py
+++ b/l10n_br_hr/models/hr_employee.py
@@ -196,7 +196,12 @@ class HrEmployee(models.Model):
 
     @api.model
     def _get_marital_status_selection(self):
-        return super()._get_marital_status_selection() + [
+        try:
+            res = super()._get_marital_status_selection()
+        except AttributeError:
+            # Fallback seguro usando a API oficial do Odoo
+            res = self.fields_get(['marital'])['marital']['selection']
+        return res + [
             ("common_law_marriage", self.env._("Common law marriage")),
             ("separated", self.env._("Separated")),
         ]
